hedera-mirror-test/k6/src/rest/test/accounts.js

```diff
@@ -0,0 +1,49 @@
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import { check } from "k6";
+import http from "k6/http";
+
+import { getOptionsWithScenario } from '../../lib/common.js';
+
+const urlTag = '/accounts';
```

Comment:
With the similar templates should we just move the main method logic to one file that each of these test js files can utilize?

The method logic I think it would be 
- the http call to get the response
- the response check with a provided text
- the status return  

```suggestion
function getAndValidateResponse(url, message) {
  const response = http.get(url);
  return check(response, {
    message: (r) => r.status === 200,
  });
}
```

Reply:
added `TestScenarioBuilder` in common.js, had to do it in the old js function way due to some weird js compatibility issue that js class doesn't work. still have to update all the rest test cases to use the builder.